.gitignore

```diff
@@ -1,6 +1,8 @@
 # Global ignores
 __pycache__/
 *.pyc
+.DS_Store
+icu4c/source/test/*/*test-*.txt
```

Comment:
These are good but unrelated changes. please move the .gitignore changes to a new PR against one of:

- https://unicode-org.atlassian.net/browse/ICU-20068
- https://unicode-org.atlassian.net/browse/ICU-20527